changelog.txt

```diff
@@ -17,6 +17,7 @@
 * Fix - Prevent empty strings being saved in related orders cache ID meta when backfilling order data to the WP Posts table.
 * Fix - Correctly load product names with HTML on the cart and checkout shipping rates.
 * Dev - Fix Node version mismatch between package.json and .nvmrc (both are now set to v16.17.1).
+* Fix - Prevent sending renewal reminders for 0 dollar orders.
```

Comment:
Small nit. Feel free to skip.

```suggestion
* Fix - Prevent sending renewal reminders for orders with a 0 total.
```

Just a small tweak to remove the reference to dollars, making the changelog entry a bit more internationally friendly.

Reply:
Actually, this changelog entry needs to move up to a new 8.2.0 one now that 8.1.0 has been deployed.